content/en/synthetics/guide/identify_synthetics_bots.md

```diff
@@ -21,23 +21,25 @@ further_reading:
 
 ## Overview
 
-Some parts of your systems might not be available to robots without the right identification, or you might want to avoid collecting analytics from Datadog robots. You can use the below methods to detect Datadog Synthetic Monitoring robots.
+Some parts of your systems may not be available to robots without the appropriate identification. You may also want to avoid collecting analytics from Datadog robots. 
+
+You can use the following methods to detect Datadog's Synthetic Monitoring robots.
```

Comment:
Relating to the comment below, do we want to highlight here that customers can use a combination of the methods for complete detection across Synthetic test configurations?

Reply:
```suggestion
Try a combination of the following methods across your Synthetic test configurations to detect Datadog's Synthetic Monitoring robots.
```